src/dotnet-new3/Program.cs

```diff
@@ -36,18 +36,6 @@ public static int Main(string[] args)
             bool disableSdkTemplates = args.Any(x => string.Equals(x, "--debug:disable-sdk-templates", StringComparison.OrdinalIgnoreCase));
             DefaultTemplateEngineHost host = CreateHost(emitTimings, disableSdkTemplates);
 
-            bool debugAuthoring = args.Any(x => string.Equals(x, "--trace:authoring", StringComparison.OrdinalIgnoreCase));
-            bool debugInstall = args.Any(x => string.Equals(x, "--trace:install", StringComparison.OrdinalIgnoreCase));
```

Comment:
I suppose this is a general cleanup and not related to post actions error reporting. Is that the case?

Reply:
it's related to https://github.com/dotnet/templating/pull/3062/commits/a958750043acba2855abcd7d403c9df44d8fc963

previously to have verbose output we have to use --trace:authoring and --trace:install managed by hack in dotnet-new3. Instead I switched to use `Reporter.Verbose` which can be enabled by using use DOTNET_CLI_CONTEXT_VERBOSE env variable. It will work for both dotnet new 3 and dotnet new

Reply:
this change is reverted